data/base/cloudreg/sle15/packages.yaml

```diff
@@ -0,0 +1,15 @@
+packages:
+  bootstrap:
+    cloudreg:
+      - cloud-regionsrv-client
+      - regionsrv-certs
```

Comment:
Revisit the inclusion of the `-release` packages here. I think they should be in a different snippet.